spec/controllers/concerns/signature_verification_spec.rb

```diff
@@ -73,6 +73,30 @@ def alternative_success
       end
     end
 
+    context 'with request older than a day' do
```

Comment:
`#matches_time_window?` uses `12.hours.ago`, spec mentions "older than a day", and the spec itself sets the header to `2.days.ago`.